tst/schedulers/bayesopt/gpautograd/test_cholesky_factorization.py

```diff
@@ -89,8 +82,20 @@ def test_cholesky_factorization():
             )
         )
         assert max_diff_grad_logdet < 1e-12
-        # mahal
-        print("mahal:")
+
+
+@pytest.mark.xdist_group("parallel")
```

Comment:
Is there a particular reason for grouping those tests in the same worker? (rather than just letting be run among any worker)

Reply:
That was done mostly to avoid resource contention. A few of your tests automatically spawn multiple cores and when they ran in parallel, they almost always ran out of the time you had allotted to them. Running them sequentially ensures that the tests which use multiple cores aren't competing among themselves.